test/service/test_app_logging.py

```diff
@@ -66,6 +66,8 @@ def test_request_logs(self):
                             'Returning 200 response with headers {"Access-Control-Allow-Origin": '
                             '"*", "Access-Control-Allow-Headers": '
                             '"Authorization,Content-Type,X-Amz-Date,X-Amz-Security-Token,X-Api-Key", '
+                            '"Content-Security-Policy": "default-src ' "'self';img-src 'self' data:;"
+                            "script-src 'self';style-src 'self';frame-ancestors 'none'" '", '
```

Comment:
This is still a little unwieldy since this one header is wrapped over two lines resulting in an odd number of internal quotes in the first and third literal.

```diff
Index: test/test_app_logging.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test/test_app_logging.py b/test/test_app_logging.py
--- a/test/test_app_logging.py	(revision 1542609bb2d275c28564278c87415d57c7e47ca4)
+++ b/test/test_app_logging.py	(date 1721349779968)
@@ -102,16 +102,23 @@
                         self.assertTrue(response.startswith(traceback_header))
                         self.assertIn(magic_message, response)
                         # … and the response is logged.
+                        headers = {
+                            'Content-Type': 'text/plain',
+                            'Content-Security-Policy': "default-src 'self';"
+                                                       "img-src 'self' data:;"
+                                                       "script-src 'self';"
+                                                       "style-src 'self';"
+                                                       "frame-ancestors 'none'",
+                            'Strict-Transport-Security': 'max-age=31536000; includeSubDomains',
+                            'X-Content-Type-Options': 'nosniff',
+                            'X-Frame-Options': 'DENY',
+                            'Cache-Control': 'no-store'
+                        }
                         self.assertEqual(
                             azul_log.output[2],
-                            'DEBUG:azul.chalice:Returning 500 response with headers {"Content-Type": "text/plain", '
-                            '"Content-Security-Policy": "default-src ' "'self';img-src 'self' data:;"
-                            "script-src 'self';style-src 'self';frame-ancestors 'none'" '", '
-                            '"Strict-Transport-Security": "max-age=31536000; includeSubDomains", '
-                            '"X-Content-Type-Options": "nosniff", '
-                            '"X-Frame-Options": "DENY", '
-                            '"Cache-Control": "no-store"}. '
-                            'See next line for the first 1024 characters of the body.\n' + response)
+                            f'DEBUG:azul.chalice:Returning 500 response with headers {json.dumps(headers)}. '
+                            f'See next line for the first 1024 characters of the body.\n' + response
+                        )
                     else:
                         # Otherwise, a generic error response is returned …
                         self.assertEqual(response.json(), {
```

As always, the refactoring (extraction of the `headers` variable) should be in a separate, early commit.